multiversx_sdk_cli/delegation/staking_provider.py

```diff
@@ -47,7 +49,13 @@ def prepare_args_for_add_nodes(args: Any):
         validator_pem = validator_pem if validator_pem.is_absolute() else validators_file_path / validator_pem
 
         secret_key_bytes, bls_key = parse_validator_pem(validator_pem)
-        signed_message = sign_message_with_bls_key(account.address.pubkey().hex(), secret_key_bytes.decode('ascii'))
+        secret_key_str = secret_key_bytes.decode()
+
+        validator_secret_key = ValidatorSecretKey.from_string(secret_key_str)
```

Comment:
An alternative (as written in the other comment) would be to use the `Message` class (which already implements `ISignable`).